raiden/raiden_service.py

```diff
@@ -371,11 +408,11 @@ def sign(self, message):
 
         message.sign(self.private_key, self.address)
 
-    def register_payment_network(self, registry_address, from_block=None):
+    def install_payment_network_filters(self, payment_network_id, from_block=None):
```

Comment:
From the usage it seems that `payment_network_id` is an address, can you add a type annotation. (for `from_block`as well pls)